lib/cli/loader.ts

```diff
@@ -216,14 +216,15 @@ export class CliLoader {
      * @private
      */
     private storeCommands(directory: string): void {
-        let manifest = getPackageManifest(directory);
+        const manifest = getPackageManifest(directory);
         if (!manifest) {
             return;
         }
 
-        let packageName = getPackageName(manifest),
-            commands = this.getCommands(directory, this.options.pattern);
-
+        const packageName = getPackageName(manifest);
+        const lscSettings = getPackageLSCSettings(manifest);
+        const pattern = (lscSettings &&  lscSettings.cliDir) ? `${lscSettings.cliDir}/*.js` : this.options.pattern;
```

Comment:
parenthesis are not required here

Reply:
makes the code more readable , it will stay like this